File: aulas_backend/java/exercicios/Premiacao.java
package aulas_backend.java.exercicios;

import java.util.Scanner;

public class Premiacao {
    public static void main(String[] args) {
        // Variáveis
        Scanner scanner = new Scanner(System.in);
        double valorCompra;
        int frequencia;

        // Solicitação de dados do usuário
        System.out.print("Digite o valor de sua compra: ");
        valorCompra = scanner.nextDouble();
        System.out.print("Digite a frequência com que compra na loja: ");
        frequencia = scanner.nextInt();

        // Verificação das condições para a premiação
        if (valorCompra >= 2000.00 && frequencia >= 10) {
            System.out.print("Parabéns! Você se tornou um cliente VIP e ganha 25% de desconto nas próximas compras!");
        } else if (valorCompra >= 1000.00 ) {
            System.out.print("Você se tornou um cliente Gold e ganha 10% de desconto nas próximas compras!");
        } else if (valorCompra >= 500.00 || frequencia >= 2) {
            System.out.print("Você se tornou um cliente Silver e ganha 5% de desconto nas próximas compras!");
        } else {
            System.out.print("Você é um cliente Bronze, contie comprando para ganhar descontos");
        }
        scanner.close();
}
}

File: atividades_linguagens_programacao/python/exercicios/premiacao.py
# Solicitação de dados ao usuário
ValorCompra = float(input("Digite o valor de suas compras: "))
Frequencia = int(input("Digite o número de vezes que você comprou na loja: "))

#  Verificação das condições para a premiação
if ValorCompra >= 2000.00 and Frequencia >= 10:
    print("Parabéns! Você se tornou um cliente VIP e ganha 25& de desconto em suas próximas compras!")
elif ValorCompra >= 1000.00:
    print("voc~e se tornou um cliente Gold e ganha 10% de desnconto em suas próximas compras!")
elif ValorCompra >= 500.00 or Frequencia >= 2:
    print("Você se tornou um cliente Silver e ganha 5% de desconto em suas próximas compras!")
else:
    print("Você é um cliente Bronze, continue comprando para ganhar descontos!")
